app/views/works/index.html.erb

```diff
@@ -0,0 +1,78 @@
+<h2>List of Works</h2>
+
+<section class="media-table">
+  <h4>Albums</h4>
+  <table class="table">
+    <thead>
+      <tr>
+        <th width="80">Votes</th>
+        <th>Title</th>
+        <th>Created By</th>
+        <th width="100">Published</th>
+        <th width="100">Upvote</th>
+      </tr>
+  </thead>
+  <tbody>
+      <% @albums.each do |album|%>
+        <tr>
+          <td><%= album.votes.count %></td>
+          <td><%= link_to "#{album.title}", work_path(album.id) %></td>
+          <td><%= album.creator %></td>
+          <td><%= album.publication_year %></td>
+          <td class="btn btn-primary"><%= link_to "Upvote", upvote_path(album.id), method: :post %><td>
+        </tr>
+      <% end %>
+  </tbody>
+</table>
+</section>
+
+<section class="media-table">
+  <h4>Books</h4>
+  <table class="table">
+    <thead>
+      <tr>
+        <th width="80">Votes</th>
+        <th>Title</th>
+        <th>Created By</th>
+        <th width="100">Published</th>
+        <th width="100">Upvote</th>
+      </tr>
+  </thead>
+  <tbody>
+      <% @books.each do |book|%>
+        <tr>
+          <td><%= book.votes.count %></td>
+          <td><%= link_to "#{book.title}", work_path(book.id) %></td>
+          <td><%= book.creator %></td>
+          <td><%= book.publication_year %></td>
+          <td class="btn btn-primary"><%= link_to "Upvote", upvote_path(book.id), method: :post %><td>
+        </tr>
+      <% end %>
+  </tbody>
+  </table>
+</section>
+
+<section class="media-table">
+<h3>Movies</h3>
+<table class="table">
+  <thead>
+    <tr>
+      <th width="80">Votes</th>
+      <th>Title</th>
+      <th>Created By</th>
+      <th width="100">Published</th>
+      <th width="100">Upvote</th>
+    </tr>
+  <tbody>
+    <% @movies.each do |movie| %>
+      <tr>
```

Comment:
You have the same code to show a list of works repeated 3 times. Could you use a view partial or a loop to DRY this up?